components/motor/fake/motor.go

```diff
@@ -97,8 +93,24 @@ type Motor struct {
 	Logger logging.Logger
 }
 
+// NewMotor creates a new fake motor.
+func NewMotor(ctx context.Context, deps resource.Dependencies, conf resource.Config, logger logging.Logger) (motor.Motor, error) {
+	logger.Error("in new motor")
+	m := &Motor{
+		Named:  conf.ResourceName().AsNamed(),
+		Logger: logger,
+		OpMgr:  operation.NewSingleOperationManager(),
+	}
+	if err := m.Reconfigure(ctx, deps, conf); err != nil {
+		return nil, err
+	}
+	logger.Errorf("new motor = %v", m)
+	return m, nil
+}
+
 // Reconfigure atomically reconfigures this motor in place based on the new config.
 func (m *Motor) Reconfigure(ctx context.Context, deps resource.Dependencies, conf resource.Config) error {
+	m.Logger.Error("in reconfigure")
```

Comment:
Same here: this is not an error. You could put it as a debug-level log, but I'm inclined to remove it again.